webrtc.html

```diff
@@ -3192,6 +3173,32 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
 
         </dl>
 
+        <dl class="idl" title="enum RTCRtpTransceiverDirection">
+          <dt>sendrecv</dt>
+          <dd>The <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpSender</code></a>
+            will offer to send RTP, and will send RTP (<var>active</var> set to "true") if the remote peer accepts.
+            The <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpReceiver</code></a> will offer to receive RTP,
+            and will receive RTP (<var>active</var> set to "true") if the remote peer accepts.</dd>
+
```

Comment:
Instead of having the parenthetical phrases (<var>active</var> set to "true"), it might be more clear with something like this:

<dd>The <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpSender</code></a> will offer to send RTP, and will send RTP if the remote peer accepts, in which case <var>active</var> is set to "true".